File: 2058-concatenation-of-array/concatenation-of-array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int>res;
        int i=0;
        while(i<nums.size())
        {
            res.push_back(nums[i]);
            i++;
        }
        i=0;
        while(i<nums.size())
        {
            res.push_back(nums[i]);
            i++;
                }
                return res;
    }
};

File: 2058-concatenation-of-array/concatenation-of-array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int ns=nums.length+nums.length;
        int res[]=new int[ns];
        int i=0;
        int n=nums.length;
        while(i<nums.length)
        {
            res[i]=nums[i];
            res[i+n]=nums[i];
            i++;
        }
        return res;
    }
}